### DISCUSSION TITLE
SpaceTeams Pro Org License Needed

### DISCUSSION THREAD

**Discussion by bishnukatuwal** [SimDynamX/SpaceTeamsPRO#44] [2024 Lunar SAR Competition Q&A] [answered]
Hello,

It is asking for a a STPro License in order to access the paid version, but all of the members of the team have a trail? Is there anyway to bypass this?

Also, getting this message on the personal use one: UE Server: Failed to connect to server, reason: [10061] Failed to connect to locally-running Compute_Server for simulation; this could mean Compute_Server crashed. Check the latest-modified file in the Compute_Server/Logs folder for more details. Would you like to return to the main menu? "No" quits the application.

**Comment by connorjak** [comment]
Please send screenshots of what you see; there may be an issue with either the app or your account. If you have accepted the trial you should be able to access with premium features by choosing `Personal Use` after login in the app.

**Reply by connorjak** [reply, marked as answer]
If you are using `Personal Use`, that is the correct way to use Space Teams for the competition. For the competition, we don't have any requirements of using the (limited for now) Org features of Space Teams PRO.

**Comment by connorjak** [comment]
See this section of the API docs for details on the crash message: https://github.com/SimDynamX/SpaceTeamsPro/wiki/STU-Lunar-SAR-API-Docs#how-to-run-the-task-sims-in-spacecraft-the-graphics-portion-of-spaceteamspro
